game_eggs/steamcmd_servers/stormworks/egg-stormworks--build-and-rescue.json

```diff
@@ -0,0 +1,54 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2022-10-26T12:10:19-06:00",
+    "name": "Stormworks: Build and Rescue",
+    "author": "iamkubi@gmail.com",
+    "description": "Join a world where you design, create and pilot your own air sea rescue service. Release your inner hero as you battle fierce storms out at sea to rescue those in need.",
+    "features": [
+        "steam_disk_space"
+    ],
+    "docker_images": {
+        "ghcr.io\/parkervcp\/yolks:wine_staging": "ghcr.io\/parkervcp\/yolks:wine_staging"
```

Comment:
Please use `Wine_staging|<url>` in the panel docker image so then it when it export it only shows wine staging to the end user.